### DISCUSSION TITLE
Quantity of devices sending logs to sensor nodes

### DISCUSSION THREAD

**Discussion by itdeveloper50** [Security-Onion-Solutions/securityonion#8883] [Unsupported Versions] [answered]
I have distributed version installed of SO and sensor nodes in my branches.

How exactly can I identify quantity of devices attached to sensors (devices that are sending logs to sensor nodes)?

**Comment by dougburks** [comment, marked as answer]
Take a look at SOC Dashboards. The main dashboard has a table for observer.name that might help:
https://docs.securityonion.net/en/2.3/dashboards.html
